src/utils/vcs.rs

```diff
@@ -237,7 +237,7 @@ fn find_reference_url(repo: &str, repos: &[Repo]) -> Result<String, Error> {
     }
 
     if found_non_git {
-        bail!("For non git repositories explicit revisions are required");
+        Ok(None)
```

Comment:
`found_`, yet return value is `Ok(None)`, it's confusing.

Reply:
True, changing it to just `non_git`.